app/src/main/java/app/foodfinderapp/LoginActivity.kt

```diff
@@ -5,18 +5,11 @@ import android.annotation.SuppressLint
 import android.content.Context
 import android.content.Intent
 import android.os.Bundle
-import android.util.Log
```

Comment:
better to remove unused imports for readability and only add the imports that are nessasary so no confusion occurs

Reply:
The IDE can (and should) organize imports automatically.  Do that, instead of manually merging a branch.
It will make no difference in runtime performance, as imports are optimized by the compiler at compile time.